src/topology/separation.lean

```diff
@@ -554,3 +554,67 @@ begin
 end
 
 end normality
+
+/-- In a compact t2 space, the connected component of a point equals the intersection of all
+its clopen neighbourhoods -/
+lemma connected_component_eq_Inter_clopen [t2_space α] [compact_space α] :
+  ∀ x : α, connected_component x = ⋂ Z : {Z : set α // is_clopen Z ∧ x ∈ Z}, Z :=
```

Comment:
instead of having a forall, can you move `x` left of the colon? i.e., have an assumption `(x : α)`? This spares an intro in the proof, and this is usually easier to read.